manifests/facts.pp

```diff
@@ -53,10 +54,39 @@
       }
     }
   } else {
-    cron{"mcollective_facts_yaml_refresh":
-      ensure  => $cron_ensure,
-      command => "'${rubypath}' '${scriptpath}' -o '${factspath}' ${factspid} &> /dev/null",
-      minute  => $cron_minutes
+    if $runmode == "cron" {
+      cron{"mcollective_facts_yaml_refresh":
+        ensure  => $cron_ensure,
+        command => "'${rubypath}' '${scriptpath}' -o '${factspath}' ${factspid} &> /dev/null",
+        minute  => $cron_minutes
+      }
+      systemd::timer { "mcollective_facts_yaml_refresh.timer":
+        active => false,
+        enable => false,
+      }
```

Comment:
This introduce a hard dependency on `puppet-systemd`, it must be added to `metadata.json`.